test/geometry/test_depth.py

```diff
@@ -329,3 +329,77 @@ def test_gradcheck(self, device, dtype):
             raise_exception=True,
             fast_mode=True,
         )
+
+
+class TestDepthFromDisparity:
+    def test_smoke(self, device, dtype):
+        disparity = torch.rand(1, 1, 3, 4, device=device, dtype=dtype)
+        baseline = torch.rand(1, device=device, dtype=dtype).item()
+        focal = torch.rand(1, device=device, dtype=dtype)
+
+        points3d = kornia.geometry.depth.depth_from_disparity(disparity, baseline, focal)
+        assert points3d.shape == (1, 1, 3, 4)
+
+    @pytest.mark.parametrize("batch_size", [2, 4, 5])
+    def test_shapes(self, batch_size, device, dtype):
+        disparity = torch.rand(batch_size, 1, 3, 4, device=device, dtype=dtype)
+        baseline = torch.rand(4, device=device, dtype=dtype)
```

Comment:
```suggestion
        baseline = torch.rand(batch_size, device=device, dtype=dtype)
```
Not very clear why 4 the size. I would assume one focal per image

Reply:
yep, batch_size would be the right input there.